lib/phobos_db_checkpoint/handler.rb

```diff
@@ -31,7 +45,15 @@ def around_consume(payload, metadata)
           end
 
           event_action = instrument('db_checkpoint.event_action', event_metadata) do
-            yield
+            begin
```

Comment:
We can add a new section to phobos.yml with the `max_retries`, something like:

```yaml
db_checkpoint:
  max_retries: 3
```

If the attribute is `nil` we keep the old behavior as the default, we keep retrying forever. If it's configured we apply the new rule, saving the failure after X retries.

WDYT? @ticolucci